src/tests/ftest/util/pool_test_base.py

```diff
@@ -96,6 +98,7 @@ def get_pool_list(self, quantity, scm_ratio, nvme_ratio):
         pool_list = [
             self.get_pool(create=False, connect=False) for _ in range(quantity)]
         for pool in pool_list:
+            pool.nsvc.update(nsvc)
```

Comment:
For whatever reason we have been using "svcn" (n in back)as the name of the our pool attributes, as opposed to the binary dmg cmd which was using "nsvc" (n in front). The pool create tests failed last time around for this reason, and probably will again until this is fixed. Option 1: You will need to do "pool.svcn.update(nsvc), or option 2 (probably better): to change the local variable name for and description also (lines 74 and and 88), then call "pool,svcn.update(svcn)" ---- OR option 3 (my personal favorite): we update the name of this property in the TestPool class to something that matches the new field name used by binary (I think you picked NumSvcResps or something) . The last option would require several changes to other tests/yaml files, Option 2 likely the easiest for now, but option 3 seems best...our naming conventions are all over the place.... @mjmac @phender

Reply:
Oops, good catch. I usually run pool/simple_create_delete.py after making changes to sanity-check, but I guess that one doesn't involve this code.